### PR TITLE
Update documentation on SLI metrics

### PR DESCRIPTION
**Pull request by logicalhan**
As per [KEP-3466](https://github.com/kubernetes/enhancements/issues/3466)

### CODE REVIEW

File: content/en/docs/reference/instrumentation/slis.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ weight: 20
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
 
 As an alpha feature, Kubernetes lets you configure Service Level Indicator (SLI) metrics
```
Comment (sftim) [inline content/en/docs/reference/instrumentation/slis.md:14]:
```suggestion
By default, Kubernetes {{< skew currentVersion >}} publishes Service Level Indicator (SLI) metrics 
```

File: content/en/docs/reference/instrumentation/slis.md [hunk 2]
```diff
@@ -9,13 +9,13 @@ weight: 20
...
-{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
 
 As an alpha feature, Kubernetes lets you configure Service Level Indicator (SLI) metrics 
 for each Kubernetes component binary. This metric endpoint is exposed on the serving 
-HTTPS port of each component, at the path `/metrics/slis`. You must enable the 
+HTTPS port of each component, at the path `/metrics/slis`. The 
 `ComponentSLIs` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-for every component from which you want to scrape SLI metrics.
+defaults to `enabled` for each Kubernetes component as of v1.27.
```
Comment (sftim) [inline content/en/docs/reference/instrumentation/slis.md:18]:
```suggestion
defaults to enabled for each Kubernetes component as of v1.27.
```
The typical values for feature gates are `true` and `false` so `enabled` doesn't seem right.

**Review by sftim — COMMENTED** [review]
Thanks

The feature gates page also needs an update for `ComponentSLIs`

File: content/en/docs/reference/instrumentation/slis.md (hunk 2 above)
Comment (logicalhan) [reply content/en/docs/reference/instrumentation/slis.md:18]: Okay will fix

**Review by sftim — COMMENTED** [review]
LGTM for docs

/lgtm

**Comment by sftim** [comment]
Ah no
> The feature gates page also needs an update for `ComponentSLIs`

/lgtm cancel

**Comment by logicalhan** [comment]
> The feature gates page also needs an update for `ComponentSLIs`

Where is that located?

**Comment by logicalhan** [comment]
Ah, found it. Updated.

**Comment by mickeyboxell** [comment]
@logicalhan could you make sure someone gives this a tech review as well?

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ For a reference to old feature gates that are removed, please refer to
 | `CSIMigrationRBD` | `false` | Alpha | 1.23 | |
 | `CSINodeExpandSecret` | `false` | Alpha | 1.25 | |
 | `CSIVolumeHealth` | `false` | Alpha | 1.21 | |
-| `ComponentSLIs` | `false` | Alpha | 1.26 | |
+| `ComponentSLIs` | `true` | Beta | 1.27 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:82]:
```suggestion
| `ComponentSLIs` | `false` | Alpha | 1.26 | 1.26 |
| `ComponentSLIs` | `true` | Beta | 1.27 | |
```

**Comment by logicalhan** [comment]
> @logicalhan could you make sure someone gives this a tech review as well?

It's reviewed.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 3 above)
Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:82]: Was this gate a thing during 1.26?
We are not supposed to eliminate the trace that the gate was Alpha during 1.26 release.
You did not fix this and you did not reply with a different opinion, but you mark this conversation resolved?

Comment (logicalhan) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:82]: Oh I thought I accepted the suggestion.

**Pull request merged**